backend/data/src/main/kotlin/io/tolgee/repository/KeyRepository.kt

```diff
@@ -204,7 +204,7 @@ interface KeyRepository : JpaRepository<Key, Long> {
     """
     from Language l
     join l.translations t
-    where t.id in (select t.id from Key k join k.translations t where k.id = :keyId)
+    where t.key.id = :keyId
```

Comment:
I feel like the previous version was doing the same as this. Am I correct? Please check